lib/Backend/Lower.cpp

```diff
@@ -6825,8 +6824,7 @@ Lowerer::GenerateStFldWithCachedType(IR::Instr *instrStFld, bool* continueAsHelp
 
     if (hasTypeCheckBailout)
     {
-        AssertMsg(PHASE_ON1(Js::ObjTypeSpecIsolatedFldOpsWithBailOutPhase) || !propertySymOpnd->IsTypeDead(),
-            "Why does a field store have a type check bailout, if its type is dead?");
+        AssertMsg(PHASE_ON1(Js::ObjTypeSpecIsolatedFldOpsWithBailOutPhase) || !propertySymOpnd->IsTypeDead(), "Why does a field load have a type check bailout, if its type is dead?");
```

Comment:
In this line and above, it looks like there's no change. Can you restore to the original version? (splitting into 2 lines will be slightly easier to read)